### ISSUE TITLE
Filters behavior switched from "eq" to "in" after commit "b4ee3fa".

### ISSUE THREAD

**Issue by hilah-ats** [Daniel-KM/Omeka-S-module-AdvancedSearch#4]
Before this commit the filters (not advanced filters) produced results exactly equal to the selection, however after this commit filters now produce results that contain the selection. This is reproduceable on a clean installation with only the Advanced Search module installed. 

Is this intended? If not, is it possible to allow configuration of the operator for filters (not advanced filters)? 

Commit in question:
https://github.com/Daniel-KM/Omeka-S-module-AdvancedSearch/commit/b4ee3fa4803ec7fa5ae361dba3e8ba1578b754ad

**Comment by hilah-ats** [comment]
Issue is still present in [3.3.6.18](https://github.com/Daniel-KM/Omeka-S-module-AdvancedSearch/releases/tag/3.3.6.18)

**Comment by Daniel-KM** [comment]
Indeed, the good behavior for filter is "and" and the optimization converted it into a list, so "or". I disabled it in last version (3.4.6.19)

**Comment by adehner** [comment]
When selecting multiple facet values for a single property, the search appears to be joined with OR instead of AND. Is this expected behavior with the internal querier?

Example:
When I select the value "Labor & Class", I get all items (count of 6) with the "Type of Social Identifier = "Labor & Class.
<img width="1173" alt="Screenshot 2023-11-13 at 2 10 56 PM" src="https://github.com/Daniel-KM/Omeka-S-module-AdvancedSearch/assets/6174191/5448313a-7b42-43c4-bacb-20250f029360">

When I add the facet value "Family Structures", I would like to see the items that have both values. Instead I see items (count of 15) that one value or the other.
<img width="1171" alt="Screenshot 2023-11-13 at 2 11 24 PM" src="https://github.com/Daniel-KM/Omeka-S-module-AdvancedSearch/assets/6174191/7430eadf-1ebc-414f-8ea8-a8bd5550a130">
